misk-redis/src/main/kotlin/misk/redis/FakeRedis.kt

```diff
@@ -158,21 +158,21 @@ class FakeRedis : Redis {
     }
   }
 
-  override fun setnx(key: String, value: ByteString) {
-    synchronized(lock) {
+  override fun setnx(key: String, value: ByteString): Boolean {
```

Comment:
Should this return 0 or 1 for consistency with the Redis docs?

Reply:
I originally had that but noticed the wrapper uses boolean for other redis commands that return 0 or 1